doc/ikfast/ikfast_tutorial.rst

```diff
@@ -73,210 +53,79 @@ Inside your catkin workspace's ``./src`` directory: ::
+  rosrun xacro xacro -o $MYROBOT_NAME.urdf $MYROBOT_NAME.urdf.xacro
 
 Select IK Type
 ^^^^^^^^^^^^^^
-You need to choose which sort of IK you want. See `this page <http://openrave.org/docs/latest_stable/openravepy/ikfast/#ik-types>`_ for more info.  The most common IK type is *transform6d*.
+You need to choose which type of IK you want to solve for. See `this page <http://openrave.org/docs/latest_stable/openravepy/ikfast/#ik-types>`_ for more info.
+The most common IK type is *transform6d*.
 
 Choose Planning Group
 ^^^^^^^^^^^^^^^^^^^^^
-If your robot has more than one arm or "planning group" that you want to generate an IKFast solution for, choose one to generate first. The following instructions will assume you have chosen one ``<planning_group_name>`` that you will create a plugin for. Once you have verified that the plugin works, repeat the following instructions for any other planning groups you have. For example, you might have 2 planning groups: ::
+If your robot has more than one arm or "planning group" that you want to generate an IKFast solver for, you need to repeat the following process for each group.
+The following instructions will assume you have chosen one ``<planning_group_name>``. Furthermore, you need to know the names of the base link and the endeffector link of the chain to solve for.
 
- <planning_group_name> = "left_arm"
- <planning_group_name> = "right_arm"
+Generate IKFast MoveIt plugin
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-To make it easy to use copy/paste for the rest of this tutorial. Set a ``PLANNING_GROUP`` environment variable. eg: ::
+To generate the IKFast MoveIt plugin, issue the following command: ::
 
- export PLANNING_GROUP="panda_arm"
-
-Identify Link Numbers
-^^^^^^^^^^^^^^^^^^^^^
-You also need the link index numbers for the ``base_link`` and ``end_effector_link`` between which the IK will be calculated. You can count the number of links by viewing a list of links in your model: ::
+  rosrun moveit_kinematics auto_create_ikfast_moveit_plugin.sh --iktype Transform6D $MYROBOT_NAME.urdf <planning_group_name> <base_link> <eef_link>
 
- openrave-robot.py "$MYROBOT_NAME".dae --info links
+The speed and success of this process will depend on the complexity of your robot. A typical 6 DOF manipulator with 3 intersecting axes at the base or wrist will take only a few minutes to generate the solver code. For a detailed explanation of the creation procedure and additional tweaks of the process, see `Tweaking the creation process`_.
 
-A typical 6-DOF manipulator should have 6 arm links + a dummy base_link as required by ROS specifications.  If no extra links are present in the model, this gives: *baselink=0* and *eelink=6*.  Often, an additional ``tool_link`` will be provided to position the grasp/tool frame, giving *eelink=7*.
+The above command creates a new ROS package named ``$MYROBOT_NAME_<planning_group_name>_ikfast_plugin`` within the current folder.
```

Comment:
"The above command" sounds awkward. It may be better to replace it with either "The command above" or "The previous command"